src/kvirc/sparser/KviIrcServerParser_literalHandlers.cpp

```diff
@@ -954,9 +954,13 @@ void KviIrcServerParser::parseLiteralPrivmsg(KviIrcMessage * msg)
 		//			}
 		//		}
 
+		// "znc.in/self-message" capability: Handle a replayed message from ourselves to someone else.
+		bool bSelfMessage = IS_ME(msg, szNick);
+		QString szWindow = bSelfMessage ? szTarget : szNick;
```

Comment:
Yes, that's what I said. In my tests I didn't see the full variant, so I assumed that ZNC never sends it. I guess it sends whatever the client sent? In either case, how can I reproduce this?